### PR TITLE
IP-2291: Improvement ID infographic interactive

### PR DESCRIPTION
**Pull request by macnguyen**

### CODE REVIEW

File: id/mapofecommerce/data/data.json [hunk 1]
```diff
@@ -813,6 +813,102 @@
       "type": "b2c",
       "category": "general",
       "location": "regional"
+    },
+    {
+      "key": 50,
+      "name": "Muslimarket",
+      "url": "https://www.muslimarket.com/",
+      "logodesktop": "logo/desktop/muslimarket.png",
```
Comment (danielbk08) [inline id/mapofecommerce/data/data.json]: Please follow name convention ex: desktopLogo, mobileLogo

Comment (macnguyen) [inline id/mapofecommerce/data/data.json]: dont like it , bro :)

File: id/mapofecommerce/index.html [hunk 2]
```diff
@@ -24,7 +24,7 @@
     <link rel="apple-touch-icon" sizes="152x152" href="https://d1vfs9f7h1rfk4.cloudfront.net/apple-touch-icon-152x152.png?v=1"/>
     <link rel="apple-touch-icon" sizes="180x180" href="https://d1vfs9f7h1rfk4.cloudfront.net/apple-touch-icon-180x180.png?v=1"/>
     <link rel="icon" type="image/png" href="https://d1vfs9f7h1rfk4.cloudfront.net/favicon-32x32.png?v=1" sizes="32x32"/>
-    <link rel="icon" type="image/png" href="https://d1vfs9f7h1rfk4.cloudfront.net/favicon-194x194.png?v=1" sizes="194x194"/>
+    <link rel="icon" type="image/png" href="https://d1vsfs9f7h1rfk4.cloudfront.net/favicon-194x194.png?v=1" sizes="194x194"/>
```
Comment (danielbk08) [inline id/mapofecommerce/index.html]: Wrong link man

Comment (macnguyen) [inline id/mapofecommerce/index.html]: changed @danielbk08

File: id/mapofecommerce/data/data.json (hunk 1 above)
Comment (danielbk08) [inline id/mapofecommerce/data/data.json]: Pls follow PSR-2 standard

Comment (macnguyen) [inline id/mapofecommerce/data/data.json]: PSR-2 only followed on PHP, we are not using for this

**Pull request merged**